### ISSUE TITLE
Is Forgejs still maintained? 

### ISSUE THREAD

**Issue by kalitine** [gopro/forgejs#174]
#### Description of the problem
Hello, I couldn't find any information on the future of this project. Is it still alive?

**Comment by Rafarel** [comment]
No, GoPro fired all the people who were working on the project. Sorry about that.

**Comment by stereo3d** [comment]
Is anybody continuing this project?

**Comment by Rafarel** [comment]
Yes, some people have forked the repo and maintained it for their own projects. There is a great chance that the code will be published any time soon.

**Comment by stereo3d** [comment]
Dear Rafarel, thank you for this good news. Is there anything we could do to encourage an update? I am especially interested in the compatibility with WebXR and Oculus. Many thanks in advance for any information.

**Comment by sgamet-vrtice** [comment]
Hi @uwe22 and @kalitine  , I am part of the team quoted by @Rafarel. We are working on this project and will publish a maintained version in coming months hopefullly. What are your expectation about a possible fork of ForgeJS ?

**Comment by stereo3d** [comment]
Dear sgamet-vrtice!

thank you for your kind update.

ForgeJS is a very great tool. I used it for teaching
interactive VR media and I made some content.
I would like to move to content creation for immersive VR or the WEBXR browser.

In the short term, I would be extremely happy if I could display the
existing content in my VR Helmet Oculus Quest in immersive view with the built in browser.
So far it is only possible to view the content in a large browser. I also added this info to github

https://lab3d.kw.ac.kr/vr/ <https://lab3d.kw.ac.kr/vr/>

In the long term I am looking for a solution where I could merge or find some bridges between ForgeJS, Blender, Armory3D and
Mozilla Hubs. I would also like to integrate this inside the E-learning software H5P which I use for my classes.

If there is anything I can do for helping or cheering you up, please let me know!

Very happy new year!

**Comment by sgamet-vrtice** [comment]
Hi @uwe22 , 

For now we are not focusing on HMD consumption. I hope it will come.
BTW I'll let you know when we'll drop an update :)

Cheers

**Comment by stereo3d** [comment]
Please cheer up for this great tool!

**Comment by stereo3d** [comment]
I hope everything goes well. Please share if there is any progress.

**Comment by stereo3d** [comment]
Best greetings, any updates maybe?

**Comment by sgamet-vrtice** [comment]
Hello @uwe22 ,
sorry I did not answer when you post your last message. We're still working on ForgeJS, but we're not focusing on HMD for the moment. We've got a lot to do before adressing this part.

**Comment by gabrielstuff** [comment]
It is super nice to read that you still maintained it.
Thanks for the work so far.

**Comment by stereo3d** [comment]
Happy New Year! Please cheer up and please keep us updated!

**Comment by stereo3d** [comment]
Please do not give up this project. Thanks for the work so far.

**Comment by sgamet-vrtice** [comment]
We did not give up @uwe22  ! Still working on this part.... slower than expected as usual !

**Comment by stereo3d** [comment]
@sgamet-vrtice hoping you are well. Could you please let us know how the project is doing? Many thanks in advance.

**Comment by sgamet-vrtice** [comment]
Hi @stereo3d !

thanks for reaching out. We made some improvements, but not related to VR / HMD features.
We've worked for people that mainly have no interest in HMD unfortunately. 

What do you expect from the "next" forgejs ?

**Comment by stereo3d** [comment]
Happy New Year @sgamet-vrtice 
It would be so wonderful to be able to use ForgeJS with a HMD and WEBXR for VR.
The immersive experience and logic is all there.
